src/autocomplete.js

```diff
@@ -490,6 +487,44 @@ export default class Autocomplete extends Component {
       inputClassList.push(inputClasses)
     }
 
+    const menuClassName = `${cssNamespace}__menu`
+    const menuModifierDisplayMenu = `${menuClassName}--${displayMenu}`
+    const menuIsVisible = menuOpen || showNoOptionsFound
+    const menuModifierVisibility = `${menuClassName}--${(menuIsVisible) ? 'visible' : 'hidden'}`
+
+    const menuClassList = [
+      menuClassName,
+      menuModifierDisplayMenu,
+      menuModifierVisibility
+    ]
+
+    if (menuClasses) {
+      menuClassList.push(menuClasses)
+    }
+
+    if (menuAttributes?.className) {
+      menuClassList.push(menuAttributes.className)
+    }
+
+    if (menuAttributes?.class) {
+      menuClassList.push(menuAttributes.class)
+    }
+
+    const computedMenuAttributes = {
+      // Copy the attributes passed as props
+      ...menuAttributes,
+      // And add the values computed for the autocomplete
+      id: `${id}__listbox`,
+      role: 'listbox',
+      className: menuClassList.join(' '),
+      onMouseLeave: this.handleListMouseLeave
```

Comment:
Removed the wrapping arrow function, as the handler is bound in the constructor already, so that'll save re-creating listeners each time the menu is re-rendered.

Reply:
Shall we do the same to `handleInputClick`?